modules/configuring-vsphere-connection-settings.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-post-installation-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-vSphere-connection-settings_{context}"]
+= Configuring the vSphere connection settings
+
+[role="_abstract"]
+Modify the following vSphere configuration settings:
+
+* vCenter network address
+* vCenter username
+* vCenter password
+* vCenter address
+* vSphere data center
+* vSphere datastore
+* Virtual machine of the cluster
+
+.Prerequisites
+* The {ai-full} has finished installing the cluster successfully.
+* The cluster is connected to link:https://console.redhat.com/.
+
+.Procedure
+. In the Administrator perspective, navigate to *Home > Overview*.
+. Under *Status*, click *vSphere connection*. This opens the *vSphere connection configuration* wizard.
+. In the *vCenter* field, enter the network address of the vSphere vCenter server. This can be either a domain name or an IP address. It appears in the vSphere web client URL; for example `https://[your_vCenter_address]/ui`. 
+. In the *Username* field, enter your vSphere vCenter username. 
+. In the *Password* field, enter your vSphere vCenter password. 
++
+[WARNING]
+====
+The system will store the username and password into the `vsphere-creds` secret name in the  `kube-system` namespace of the cluster. An incorrect vCenter username or password will make the cluster nodes unschedulable.
```

Comment:
`vsphere-creds` is the secret name, suggest to change 
`The system will store the username and password into the `vsphere-creds` secret name`
to
`The system will store the username and password into the `vsphere-creds` secret`